package.json

```diff
@@ -35,7 +35,7 @@
     "jquery-ui-npm": "1.12.0",
     "marked": "^0.3.17",
     "minimist": "^1.2.0",
-    "nw": "^0.50.3-sdk",
+    "nw": "0.53.1-sdk",
```

Comment:
TBH we do not need sdk flavor of nw and this might go only for 3.1
No major component upgrade between RC

Reply:
And see my comment under the PR

Reply:
Got it. In this case I will close the PR